apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -340,6 +340,11 @@ type ClusterResourcePlacementStatus struct {
 	// +optional
 	SelectedResources []ResourceIdentifier `json:"selectedResources,omitempty"`
 
+	// ClusterResourceSnapshots is a list of clusterResourceSnapshot names.
```

Comment:
Does this only contain the most recent resource snapshots or all existing resource snapshots for this placement?

Reply:
I assume this only contains the current resource snapshots.

Reply:
If so, be explicit in the comment.